### ISSUE TITLE
Error: Cannot find module 'parse5' on npm start

### ISSUE THREAD

**Issue by prashant1587** [angular/universal-starter#364]
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /home/pbadmin/universal-starter/dist/**/* src/index.html
[nodemon] starting `node dist/server/index.js`
module.js:327
    throw err;
    ^

Error: Cannot find module 'parse5'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/pbadmin/universal-starter/dist/server/index.js:41884:18)
    at __webpack_require__ (/home/pbadmin/universal-starter/dist/server/index.js:27:30)
    at Object.<anonymous> (/home/pbadmin/universal-starter/dist/server/index.js:40698:14)
    at __webpack_require__ (/home/pbadmin/universal-starter/dist/server/index.js:27:30)
    at Object.<anonymous> (/home/pbadmin/universal-starter/dist/server/index.js:54367:10)
    at __webpack_require__ (/home/pbadmin/universal-starter/dist/server/index.js:27:30)
[nodemon] app crashed - waiting for file changes before starting...

**Comment by prashant1587** [comment]
Thanks...
It was indeed a node version issue.
After I migrated to node version 7 it worked

**Comment by MarkPieszak** [comment]
Hey we've updated this starter to include a `CLI` and a standalone webpack example using the latest `@angular/platform-server` that's been moved into Core. Closing this out since it's relating to the previous version, angular2-universal, but feel free to open a new issue if there's still an issue with the new version! Thanks.
